src/main/java/com/direwolf20/buildinggadgets/common/tools/ExchangingModes.java

```diff
@@ -45,17 +172,11 @@ private static boolean isReplaceable(World world, BlockPos pos, IBlockState curr
         if (worldBlockState.getMaterial() == Material.AIR) {
             return false;
         }
-        if (worldBlockState.getMaterial().isLiquid()) {
-            return false;
-        }
-        /*if (world.getBlockState(pos) != currentBlock || world.getBlockState(pos) == ModBlocks.effectBlock.getDefaultState() || world.getBlockState(pos) == setBlock || world.getTileEntity(pos) != null || currentBlock.getBlock().getBlockHardness(currentBlock, world, pos) < 0) {
-            return false;
-        }*/
-        return true;
+        return !worldBlockState.getMaterial().isLiquid();
     }
 
     public static void addConnectedCoords(World world, BlockPos loc, IBlockState state, IBlockState setBlock,
-            boolean fuzzyMode, List<BlockPos> coords, int minX, int minY, int minZ, int maxX, int maxY, int maxZ) {
+                                          boolean fuzzyMode, List<BlockPos> coords, int minX, int minY, int minZ, int maxX, int maxY, int maxZ) {
```

Comment:
What happened here! haha :P

Reply:
+ Do not align parameters when they are multi-line